### PR TITLE
make gc docs referencable by lakefs cloud docs

### PR DESCRIPTION
**Pull request by talSofer**
1. Add a TOC so that it easy to point to the docs page from lakeFS cloud docs 
2. Add docs for how to create GC rules from the UI

### CODE REVIEW

File: docs/reference/garbage-collection.md [hunk 1]
```diff
@@ -62,6 +68,17 @@ EOT
 lakectl gc set-config lakefs://example-repo -f example_repo_gc_rules.json 
 ```
 
+### From the lakeFS UI
+{: .no_toc }
+
+1. Click on the repository you would like to configure GC rules for
+2. Go to _Settings_ -> _Retention_
+3. Click _Edit policy_ and paste your GC rule into the text box as Json
+4. Save you changes
```
Comment (johnnyaug) [inline docs/reference/garbage-collection.md:77]:
```suggestion
1. Naivgate to the main page of your repository.
2. Go to _Settings_ -> _Retention_.
3. Click _Edit policy_ and paste your GC rule into the text box as a JSON.
4. Save your changes.
```

**Review by johnnyaug — APPROVED** [review]
LGTM, Small suggestion

File: docs/reference/garbage-collection.md (hunk 1 above)
Comment (talSofer) [reply docs/reference/garbage-collection.md:77]: done thanks

**Review by arielshaqed — APPROVED** [review]
Thanks!

**Review by Jonathan-Rosenberg — APPROVED** [review]
Looks cool, just mind the Grammarly icon in the image...

**Pull request merged**